middleware/user.update.js

```diff
@@ -0,0 +1,26 @@
+const joi = require('joi');
+const logger = require('../utils/logger');
+
+const validateUpdateUser = async (req, res, next) => {
+  const schema = joi.object({
+    first_name: joi.string(),
+    last_name: joi.string(),
+    email: joi.string(),
+    phone: joi.string().optional(),
```

Comment:
We either remove the `optional` flag completely or set it on all the fields, since they are optional by default